### ISSUE TITLE
Error when reco start and selecting 2nd option

### ISSUE THREAD

**Issue by wizevlad** [orchoban/react.cordova#4]
I want to run the recently created app into the ios emulator:
```
reco start
-> selected 2nd option
```
gives this error:
```
reco-cli-serve ERROR : cordova serve error ,RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
```

**Comment by wizevlad** [comment]
Regardless of the above error, what's the recommended way of developing and see changes in both the browser and the ios simulator?
I need this because I'm working with cordova plugins and need to see the changes in the simulator as I code.

thanks in advance

**Comment by orchoban** [comment]
Hey, I see the error.  
I'll deal with that soon.

**Comment by orchoban** [comment]
And about your question.  I recommend working on React Simulator (the first option in `reco start`) or on the Cordova Browser Simulator (Option 2 in `reco start`). you can open `reco start` on Safari to feel the ios Webview).
After that, try to run it on the Android Simulator. 
 Only at the end will you check on an iPhone simulator

**Comment by orchoban** [comment]
@wizevlad 
I fixed the problem. 
Available from Version 1.7.3
